### PR TITLE
Fix typos in CSV::Row docs

### PR DESCRIPTION
**Pull request by vbrazo**
- Fix typos in `CSV::Row` documentation
- Remove unnecessary spaces

### CODE REVIEW

File: lib/csv/row.rb [hunk 1]
```diff
@@ -172,8 +172,8 @@ def []=(*args)
     #   <<( header_and_field_hash )
     #
     # If a two-element Array is provided, it is assumed to be a header and field
-    # and the pair is appended.  A Hash works the same way with the key being
-    # the header and the value being the field.  Anything else is assumed to be
+    # and the pair is appended. A Hash works the same way with the key being
+    # the header and the value being the field. Anything else is assumed to be
     # a lone field which is appended with a +nil+ header.
```
Comment (vbrazo) [inline lib/csv/row.rb]: ```Anything else is assumed to be a lone field which is appended with a +nil+ header.```

Is `lone field` correct? I was wondering if it should be `line field`.

File: lib/csv/row.rb [hunk 2]
```diff
@@ -13,13 +13,13 @@ class CSV
...
     #
-    # Construct a new CSV::Row from +headers+ and +fields+, which are expected
-    # to be Arrays.  If one Array is shorter than the other, it will be padded
+    # Constructs a new CSV::Row from +headers+ and +fields+, which are expected
+    # to be Arrays. If one Array is shorter than the other, it will be padded
     # with +nil+ objects.
     #
     # The optional +header_row+ parameter can be set to +true+ to indicate, via
     # CSV::Row.header_row?() and CSV::Row.field_row?(), that this is a header
-    # row.  Otherwise, the row is assumes to be a field row.
```
Comment (vbrazo) [inline lib/csv/row.rb]: grammar issue

File: lib/csv/row.rb (hunk 1 above)
Comment (kou) [reply lib/csv/row.rb]: `lone field` is correct: "being alone" in https://www.dictionary.com/browse/lone

But we can use other wording.

Comment (vbrazo) [reply lib/csv/row.rb]: gotcha! Thanks for the link @kou 👍

**Pull request merged**